public/configs/exchanges.json

```diff
@@ -0,0 +1,8 @@
+[
+  {
+      "name": "coinbase",
+      "label": "Coinbase",
+      "icon": "/exchanges/coinbase.png",
```

Comment:
nit: I wonder if we should be following the pattern of mapping label/symbol to path in `AssetIcon` (like we do with markets right now)

Reply:
This config would also used by Native clients, and the would need to fetch the icons from the public directory the web hosts. So it would need to be the actual URL path to those assets